src/tasks/translations.cpp

```diff
@@ -69,6 +69,17 @@ translations::projects::lang::lang(std::string n)
 {
 }
 
+std::pair<std::string, std::string> translations::projects::lang::split() const
+{
+	const auto p = name.find('_');
```

Comment:
Wouldn't it be more safe to `rfind` in case a project name contains underscores? I don't think the language "tag" can contain underscore.

Reply:
This isn't the project name, it's just the language name. It's basically the file name without extension from `transifex-translations/translations/project name/`, like `de.ts`, `fr.ts`, `pt_BR.ts`, etc.

Reply:
Ah, yes, I did not notice it was in the inner-class.